revenuecatui/revenuecatui.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |spec|
 
         Alternatively, proper pod installation is performed during Gradle sync in the IDE (if Podfile location is set)"
     end
-
+                
```

Comment:
```suggestion

```

Reply:
This was done automatically by `pod` (I ran `pod repo update` because Gradle told me to). I was thinking to just commit it, otherwise we keep having to "fight" it by reverting these changes. (I'm assuming `pod` is consistent in what it wants in terms of newlines.)

Reply:
oh weird, leave it as is then!